modules/upnp/doc_classes/UPNP.xml

```diff
@@ -67,9 +74,11 @@
 			<param index="3" name="proto" type="String" default="&quot;UDP&quot;" />
 			<param index="4" name="duration" type="int" default="0" />
 			<description>
-				Adds a mapping to forward the external [code]port[/code] (between 1 and 65535) on the default gateway (see [method get_gateway]) to the [code]internal_port[/code] on the local machine for the given protocol [code]proto[/code] (either [code]TCP[/code] or [code]UDP[/code], with UDP being the default). If a port mapping for the given port and protocol combination already exists on that gateway device, this method tries to overwrite it. If that is not desired, you can retrieve the gateway manually with [method get_gateway] and call [method add_port_mapping] on it, if any.
+				Adds a mapping to forward the external [code]port[/code] (between 1 and 65535, although recommended to use port 1024 or above) on the default gateway (see [method get_gateway]) to the [code]internal_port[/code] on the local machine for the given protocol [code]proto[/code] (either [code]TCP[/code] or [code]UDP[/code], with UDP being the default). If a port mapping for the given port and protocol combination already exists on that gateway device, this method tries to overwrite it. If that is not desired, you can retrieve the gateway manually with [method get_gateway] and call [method add_port_mapping] on it, if any. Note that forwarding a well-known port (below 1024) with UPnP may fail depending on the device.
```

Comment:
Could update the docs to use `[param]`.

I'd suggest also renaming `desc` and `proto` to avoid abbreviations.

Reply:
I left `[param]` out so it can be cherry-picked to 3.x more easily.

I want to do some followups for UPnP that will be 4.0/4.x only, and it probably should be moved to a extension too. I can do both there, or I guess I can do the `desc` and `proto` changes here, I assume we don't consider changing param names to be compat breaking?

Reply:
Yeah I thought about that too. I guess it's fine as is for a 3.x backlog and further changes can be made for 4.0.